### PR TITLE
Crud logs hibernate

### PR DESCRIPTION
**Pull request by igorkraskovskiii**

### CODE REVIEW

File: src/main/java/app/resources/LogsResource.java [hunk 1]
```diff
@@ -17,49 +21,55 @@
...
     }
 
     @GET
+    @Path("/{logsId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Log get(@PathParam("logsId") int logsId) {
+        return basicCrudDao.findById(logsId);
+    }
+
+/*    @GET
```
Comment (AlexKolonitsky) [inline src/main/java/app/resources/LogsResource.java]: remove comment

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/resources/LogsResource.java (hunk 1 above)
Comment (igorkraskovskiii) [reply src/main/java/app/resources/LogsResource.java]: made

File: pom.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
         <hibernate.version>5.4.2.Final</hibernate.version>
 
         <jdbc.driver>com.mysql.cj.jdbc.Driver</jdbc.driver>
-        <db.url>jdbc:mysql://localhost:3306/timesheet_dev?useUnicode=true&amp;useJDBCCompliantTimezoneShift=true&amp;useLegacyDatetimeCode=false&amp;serverTimezone=UTC</db.url>
+        <db.url>jdbc:mysql://localhost:3306/timesheetmanager</db.url>
```
Comment (AlexKolonitsky) [inline pom.xml]: remove changes

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/test/resources/input.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version= '1.0' encoding= 'UTF-8'?>
+<dataset>
+    <logs assignmentId="1" time="1.0" order="1" comment="qwe" date="2019-12-03"/>
+   <!-- <assignment id="1" employeeId="1" projectId="1" activityId="1" workLoad="1"/>
+    <activity id="1" name="serg"/>
```
Comment (AlexKolonitsky) [inline src/test/resources/input.xml]: remove comments

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (igorkraskovskiii) [reply pom.xml]: fixed

Comment (igorkraskovskiii) [reply pom.xml]: fixed

File: src/test/resources/input.xml (hunk 3 above)
Comment (igorkraskovskiii) [reply src/test/resources/input.xml]: fixed

File: pom.xml [hunk 4]
```diff
@@ -149,19 +149,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```
Comment (AlexKolonitsky) [inline pom.xml:152]: why you remove slf4j ?

**Review by AlexKolonitsky — APPROVED** [review]

**Pull request merged**